src/Tickets/Seating/Admin/Ajax.php

```diff
@@ -1017,6 +1017,18 @@ public function handle_seat_type_deleted() {
 		global $wpdb;
 
 		try {
+			$original_seat_types_tickets = array_map(
+				'intval',
+				DB::get_col(
+					DB::prepare(
+						'SELECT DISTINCT(post_id) FROM %i WHERE meta_key = %s AND meta_value = %s',
```

Comment:
I would modify this to `JOIN` on the `posts` table and include the post type in the query to make sure you're getting only the post IDs of tickets.

You can pull the ticket post types using the `tribe_tickets()->ticket_types()` function to build a comma-separated list.

Right now, the only entity that can be assigned post types are tickets, but that could change due to future changes or user-land modifications.

In general: always query by post type when querying posts.

Reply:
Agreed. makes sense. Fixing the update query as well based on your comment!

Reply:
Correction: Seems like i shouldn't change the update query just below because attendee's meta is expected to be uddated as well.